javascript/src/components/inputs/InputIOMetrics.jsx

```diff
@@ -3,22 +3,33 @@
 var React = require('react');
 //noinspection JSUnusedGlobalSymbols
 var MetricsStore = require('../../stores/metrics/MetricsStore');
+var NodesStore = require('../../stores/nodes/NodesStore');
 var numeral = require('numeral');
 
 var metricsStore = MetricsStore.instance;
+var nodesStore = NodesStore.instance;
 
 var InputIOMetrics = React.createClass({
     getInitialState() {
         return {
             initialized: false,
             hasError: false,
+            showDetails: false,
+            global: this._newMetricState(),
+            nodes: {}
```

Comment:
Better to use a Map or Set here: http://facebook.github.io/immutable-js/docs/#/Map